decimalSuite_test.go

```diff
@@ -100,15 +97,15 @@ func convertToDec64(testvals testCaseStrings) (dec64Vals decValContainer) {
 }
 
 // runTest completes the tests and returns a boolean and string on if the test passes
-func runTest(testVals decValContainer, testValStrings testCaseStrings) (testFailed bool, testString string) {
+func runTest(testVals decValContainer, testValStrings testCaseStrings) (testPassed bool, testString string) {
```

Comment:
D'oh, why didn't I notice before! Your string is basically an error message. Instead of a bool and a string, you should just return an `error` (`nil` if it passed).

Sorry, I should have noticed this earlier.